_resources/port1.0/fetch/mirror_sites.tcl

```diff
@@ -530,218 +470,213 @@ if {${distfiles_scheme} eq "https"} {
-    http://mirrors.ukfast.co.uk/sites/ftp.mysql.com/Downloads/:nosubdir
-    http://mirrors.xservers.ro/mysql/Downloads/:nosubdir
-    http://mysql.he.net/Downloads/:nosubdir
-    http://mysql.infocom.ua/Downloads/:nosubdir
-    http://mysql.inspire.net.nz/Downloads/:nosubdir
-    http://mysql.linux.cz/Downloads/:nosubdir
-    http://mysql.mirror.ac.za/Downloads/:nosubdir
-    http://mysql.mirror.kangaroot.net/Downloads/:nosubdir
-    http://mysql.mirrors.arminco.com/Downloads/:nosubdir
-    http://mysql.mirrors.crysys.hit.bme.hu/Downloads/:nosubdir
-    http://mysql.mirrors.hoobly.com/Downloads/:nosubdir
-    http://mysql.mirrors.ovh.net/ftp.mysql.com/Downloads/:nosubdir
-    http://mysql.mirrors.pair.com/Downloads/:nosubdir
-    http://mysql.spd.co.il/Downloads/:nosubdir
-    http://na.mirror.garr.it/mirrors/MySQL/Downloads/:nosubdir
-    http://sunsite.icm.edu.pl/mysql/Downloads/:nosubdir
-    http://www.linorg.usp.br/mysql/Downloads/:nosubdir
-    https://www.mirrorservice.org/sites/ftp.mysql.com/Downloads/:nosubdir
+    https://dev.mysql.com/get/Downloads/:nosubdir
 }
 
+# https://www.netbsd.org/mirrors/
+# CDN: http://cdn.NetBSD.org/pub/NetBSD/
 set portfetch::mirror_sites::sites(netbsd) {
-    https://ftp.NetBSD.org/pub/NetBSD/
-    http://ftp7.de.NetBSD.org/pub/ftp.netbsd.org/pub/NetBSD/
+    ftp://ftp.NetBSD.org/pub/NetBSD/
+    http://ftp.NetBSD.org/pub/NetBSD/
+    http://cdn.NetBSD.org/pub/NetBSD/
+    ftp://ftp.dk.NetBSD.org/pub/NetBSD/
+    ftp://ftp.ee.NetBSD.org/pub/NetBSD/
+    ftp://ftp.fi.NetBSD.org/pub/NetBSD/
+    http://ftp.fi.NetBSD.org/pub/NetBSD/
+    ftp://ftp2.fr.NetBSD.org/pub/NetBSD/
     http://ftp.fr.NetBSD.org/pub/NetBSD/
+    ftp://ftp4.fr.NetBSD.org/mirrors/ftp.netbsd.org/
+    ftp://ftp.de.NetBSD.org/pub/NetBSD/
+    ftp://ftp3.de.NetBSD.org/pub/NetBSD/
+    ftp://ftp.hu.NetBSD.org/pub/NetBSD/
+    ftp://ftp2.ie.NetBSD.org/mirrors/ftp.netbsd.org/pub/NetBSD/
+    http://mirror.isoc.org.il/pub/netbsd/
+    ftp://ftp2.jp.NetBSD.org/pub/NetBSD/
+    ftp://ftp6.jp.NetBSD.org/pub/NetBSD/
     ftp://ftp7.jp.NetBSD.org/pub/NetBSD/
-    ftp://ftp.ru.NetBSD.org/pub/NetBSD/
+    ftp://ftp4.jp.NetBSD.org/pub/NetBSD/
+    ftp://ftp.kaist.ac.kr/NetBSD/
+    ftp://ftp.nl.NetBSD.org/pub/NetBSD/
+    ftp://mirrors.up.pt/pub/NetBSD/
     ftp://ftp.tw.NetBSD.org/pub/NetBSD/
-    ftp://ftp.uk.NetBSD.org/pub/NetBSD/
-    ftp://ftp7.us.NetBSD.org/pub/NetBSD/
+    ftp://mirror.planetunix.net/pub/NetBSD/
+    http://mirror.planetunix.net/pub/NetBSD/
+    http://netbsd.mirrors.hoobly.com/
 }
 
 # Equivalent to "savannah"; neither name takes precedence over the other.
 # https://download-mirror.savannah.gnu.org/releases/00_MIRRORS.txt
 set portfetch::mirror_sites::sites(nongnu) {
     http://mirror.csclub.uwaterloo.ca/nongnu/
     ftp://mirror.csclub.uwaterloo.ca/nongnu/
-    http://babyname.tips/mirrors/nongnu/
+    http://nongnu.freemirror.org/nongnu/
+    ftp://nongnu.freemirror.org/nongnu
```

Comment:
Don't list both the ftp and the http version of the same server; just list the http version, since http works for almost everybody while ftp is frequently problematic.

Reply:
Does the same policy apply to dupes of ftp vs http**s**? Or is ftp preferred over https, for older systems? ie…
ftp > https?
https > ftp?

Reply:
I hadn't thought about it... I think my preference in that case is for https over ftp. I just don't like ftp very much. When it works, great, but when it doesn't, the connection can hang. At least https should give an immediate answer: either 200 success or 404 not found or an SSL error.